posthog/session_recordings/queries/session_recording_list_from_filters.py

```diff
@@ -473,7 +474,7 @@ def _where_predicates(self) -> ast.Expr:
             exprs.append(
                 ast.CompareOperation(
                     op=ast.CompareOperationOp.In,
-                    left=ast.Constant(value="`$session_id`"),
+                    left=ast.Field(chain=["$session_id"]),
```

Comment:
should we have a test? (what's the error?)

Reply:
It was using the constant (string value) not the field e.g. `"$session_id" in ['uuid1', 'uuid2',...]` instead of `$session_id in ['uuid1', 'uuid2',...]`

I think the effect is that pinned recordings weren't being shown in playlists for HogQL filters.

One of my new tests relies on the `$session_id` field so that should cover it